atom/browser/api/atom_api_desktop_capturer.cc

```diff
@@ -220,13 +224,18 @@ void DesktopCapturer::BuildPrototype(
 
 namespace {
 
+mate::Handle<atom::api::DesktopCapturer> CreateDesktopCapturer(
+    v8::Isolate* isolate) {
+  return atom::api::DesktopCapturer::Create(isolate);
+}
+
 void Initialize(v8::Local<v8::Object> exports,
                 v8::Local<v8::Value> unused,
                 v8::Local<v8::Context> context,
                 void* priv) {
   v8::Isolate* isolate = context->GetIsolate();
   mate::Dictionary dict(isolate, exports);
-  dict.Set("desktopCapturer", atom::api::DesktopCapturer::Create(isolate));
+  dict.SetMethod("createDesktopCapturer", &CreateDesktopCapturer);
```

Comment:
```suggestion
  dict.SetMethod("createDesktopCapturer", &atom::api::DesktopCapturer::Create);
```

no need for a helper function